lambda_functions/transform_cloudwatch_lambda.py

```diff
@@ -0,0 +1,147 @@
+import boto3
+import gzip
+import json
+import os
+import logging
+from functools import lru_cache
+import base64
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+default_keys_to_remove = ["account_id", "region"]
+
+
+def lambda_handler(event, context):
+    """
+    This function processes CloudWatch Logs from Firehose.
+    """
+    output_records = []
+    try:
+        rds_prefix = make_prefixes()
+        region = boto3.Session().region_name or os.environ.get("AWS_REGION")
+        account_id = os.environ.get("ACCOUNT_ID")
+        if not account_id:
+            raise ValueError("ACCOUNT_ID environment variable is reuqired")
+
+        rds_client = boto3.client("rds", region_name=region)
+    except Exception as e:
+        logger.error(f"Initialization error: {str(e)}")
+        # this will store the records in the s3 bucket as untransformable for later retrying
+        return {"records", []}
+
+    try:
+        for record in event["records"]:
+            pre_json_value = gzip.decompress(base64.b64decode(record["data"]))
+            processed_logs = []
+            for line in pre_json_value.strip().splitlines():
+                logs = json.loads(line)
+                for key in default_keys_to_remove:
```

Comment:
Why do we have to remove these keys?

Reply:
it will never affect it because we build the logs anyway, removed